src/app/Login/page.tsx

```diff
@@ -0,0 +1,91 @@
+"use client"
+import { useState } from "react"
+import { Container, Typography, TextField, Button, Snackbar } from "@mui/material"
+import { useNavigate } from "react-router-dom"
+import './Login.css'
+
+function Login() {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [error, setError] = useState<string | null>(null)
+    const [openSnackbar, setOpenSnackbar] = useState<boolean>(false)
```

Comment:
```suggestion
    const [openSnackbar, setOpenSnackbar] = useState(false)
```
boolean같은 원시 타입은 타입 명시 안 해줘도 되는 것으로 알고 있습니다